src/wp-admin/about.php

```diff
@@ -246,7 +265,7 @@
 						printf(
 							/* translators: %s: WordPress version number. */
 							__( 'WordPress %s Field Guide' ),
-							'6.7'
+							$display_version
```

Comment:
Same for field guide.

```suggestion
							'6.8'
```